app/controllers/support_users/api_clients_controller.rb

```diff
@@ -0,0 +1,36 @@
+class SupportUsers::ApiClientsController < ApplicationController
```

Comment:
This is publicly accessible and doesn't require authentication. 

![image](https://github.com/user-attachments/assets/51062879-0e0c-4e6a-be0f-43c50aede000)